### DISCUSSION TITLE
Building: Package Request: cccc #9829

### DISCUSSION THREAD

**Discussion by shoya0x00** [termux/termux-packages#9836] [Q&A]
#9829

No configuration file only makefile. How can I skip the configuration process

**Comment by Grimler91** [comment]
It is skipped automatically. You will need to set `TERMUX_PKG_BUILD_IN_SRC=true` to make the `termux_step_make` step find the Makefile though

**Reply by shoya0x00** [reply]
#9839 

Any idea what I am missing. On Device Build is okay. It has no issue while building

**Comment by 2096779623** [comment]
```shell
../pccts/bin/antlr -CC -k 2 -gd -ge -rl 5000 -w1 -e3  -ft Ctokens.h cccc.g
../pccts/bin/antlr: 2: Syntax error: "(" unexpected
make[1]: *** [rules.mak:181: cccc.cpp] Error 2
make[1]: Leaving directory '/home/builder/.termux-build/cccc/src/cccc'
make: *** [makefile:44: cccc] Error 2
Error: Process completed with exit code 2.
```

**Reply by shoya0x00** [reply]
On device don't give any error. Why is the builder giving error
